test/scripts/imgtestlib.py

```diff
@@ -155,6 +179,30 @@ def check_config_names():
         sys.exit(1)
 
 
+def gen_manifests(outputdir, config_map=None, distros=None, arches=None, images=None,
+                  commits=False, skip_no_config=False):
+    # pylint: disable=too-many-arguments
+    cmd = ["go", "run", "./cmd/gen-manifests",
+           "--cache", os.path.join(TEST_CACHE_ROOT, "rpmmd"),
+           "--output", outputdir,
+           "--workers", "100"]
+    if config_map:
+        cmd.extend(["--config-map", config_map])
+    if distros:
+        cmd.extend(["--distros", ",".join(distros)])
+    if arches:
+        cmd.extend(["--arches", ",".join(arches)])
+    if images:
+        cmd.extend(["--images", ",".join(images)])
+    if commits:
+        cmd.append("--commits")
+    if skip_no_config:
+        cmd.append("--skip-noconfig")
+    print("⌨️" + " ".join(cmd))
+    _, err = runcmd(cmd, extra_env=rng_seed_env())
```

Comment:
(super nitpick) when I read this for the first time I was slightly confused and assume `err` is a go style error value. But it's `stderr` so *maybe* call it srderr the easily confused :) ?

Reply:
Meh, this should read ` so maybe call it srderr *to help* the easily confused` (I guess it just proves a point about me being easily confused)